edge/evtstreams/cpu2evtstreams/CreateService.md

```diff
@@ -60,65 +60,67 @@ Follow the steps in this page to create your own CPU To IBM Event Streams Edge S
 ```bash
 cd ~   # or wherever you want
 git clone git@github.com:open-horizon/examples.git
+cd examples/
 ```
 
-2. Checkout the branch that corresponds to your horizon CLI version. To get the branch name, remove the last bullet and any numbers after it, then prepend a `v` at the beginning:
-```bash
-$ hzn version
-Horizon CLI version: 2.27.0-173 # Branch name in this example is v2.27
-Horizon Agent version: 2.27.0-173
-$ git checkout v2.27
-```
+2. Check your Horizon CLI version:
+
+  ```bash
+  hzn version
+  ```
+
+3. Starting with Horizon version `v2.29.0-595` you can `checkout` to a version of the example services that directly corresponds to your Horizon CLI version with these commands: 
```

Comment:
Should we leave the old instructions in case people aren't using Horizon version `v2.29.0-595`?

Reply:
I like the idea of covering all the bases, but I'd like to at least rephrase it so and remove all the commands... what do you think of something like the following? 

```
Note: if you are using an older version of the hzn CLI you can checkout to the branch that corresponds to the major version you are using. For example Horizon CLI version: 2.27.0-173 can use git checkout v2.27 
```

Reply:
Sounds good to me